src/components/Tables/MeetingVotesTableRow/MeetingVotesTableRow.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import DecisionResult from "../../Shared/DecisionResult";
+import { MATTER_STATUS_DECISION } from "../../../constants/ProjectConstants";
+import { IndividualMeetingVote } from "../../Shared/Types/IndividualMeetingVote";
+import { VOTE_DECISION } from "../../../constants/ProjectConstants";
+import { STYLES } from "../../../constants/StyleConstants";
+import { Link } from "react-router-dom";
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+type MeetingVotesTableRowProps = {
+  /** the name of the matter that was voted on */
+  legislationName: string;
+  /** the index of the row */
+  index: number;
+  /** the voting body decision */
+  councilDecision: MATTER_STATUS_DECISION;
+  /** link to the detail page of the matter being voted on */
+  legislationLink: string;
+  /** date of the matter being voted on */
+  meetingDate: string;
+  /** vote results by individual members on the matter in this row */
+  votes: IndividualMeetingVote[];
+};
+
+function renderVotesCell(isExpanded: boolean, votes: IndividualMeetingVote[]) {
+  if (isExpanded) {
+    return (
+      <td>
+        {votes.map((vote, index) => {
+          return (
+            <div
+              key={`individual-vote-${index}-${vote.id}`}
+              style={{
+                display: "flex",
+                flexDirection: "row",
+                justifyContent: "center",
+                alignItems: "center",
+              }}
+            >
+              <DecisionResult result={vote.decision} />
+              <p style={{ flex: 1 }}>{vote.name}</p>
+            </div>
+          );
+        })}
+      </td>
+    );
+  } else {
+    let votesFor = 0;
+    let votesAgainst = 0;
+    let votesAbstained = 0;
+    votes.forEach((vote) => {
+      if (vote.decision === VOTE_DECISION.APPROVE) votesFor++;
+      if (vote.decision === VOTE_DECISION.REJECT) votesAgainst++;
+      if (vote.decision === VOTE_DECISION.ABSTAIN) votesAbstained++;
+    });
+    const votesMinified = `${votesFor} Approved   /   ${votesAgainst} Rejected   /   ${votesAbstained} Abstained`;
```

Comment:
Could we use the `TAG_CONNECTOR` constant: https://github.com/CouncilDataProject/cdp-frontend/blob/main/src/constants/StyleConstants.ts#L1 instead of `/` and can we make those tabs? are they tabs? should we use the html code for a tab? Just to make the spacing consistent because the numerics have minor but like 1px differences in width :joy:

Reply:
Looking at both options I actually don't mind the slashes all that much, although both look great so I'm not too opinionated on this one. 

I will agree that within the code the spacing should be changed to a single space versus a tab but the tab spacing gets collapsed anyway and the tab spacing strays from all the other string template literals we have in the CDP codebase.

<img width="417" alt="Screen Shot 2021-05-01 at 6 22 54 PM" src="https://user-images.githubusercontent.com/39639992/116799032-4bee4200-aaaa-11eb-9170-e042308306c5.png">
<img width="413" alt="Screen Shot 2021-05-01 at 6 22 46 PM" src="https://user-images.githubusercontent.com/39639992/116799031-4a247e80-aaaa-11eb-8bd8-3e03469b4ab3.png">

Reply:
I am a fan of the dots instead of slashes for consistency and because I don't want there to be _any_ confusion about "wait its 4 approvals OR 3 rejections OR 2 abstentions???"

Reply:
In terms of spacing, yea just make it a single space probably.

Reply:
Ooh okay I can behind that reasoning.